workspace/workflows/jobs.py

```diff
@@ -14,6 +15,7 @@
 from workspace.workflows import config
 
 
+CACHE_PATH = settings.APPLICATION_ROOT / "workspace" / "workflows" / "cache.json"
```

Comment:
This will need to be a path in `settings.WRITEABLE_DIR` (in production, the docker user doesn't have permission to write inside the container; plus, this way the cache file isn't lost when the container is restarted)

Reply:
I see! I have called it workflows_json and put it in `settings.WRITEABLE_DIR`